README.md

```diff
@@ -8,22 +8,12 @@ Fleet is the lightweight, open source telemetry platform for servers and worksta
 
 ## Try Fleet
 
-#### With [Node.js](https://nodejs.org/en/download/) and [Docker](https://docs.docker.com/get-docker/) installed:
-
-```bash
```

Comment:
Who is the audience for README.md? Is it people looking at our repo on github.com? If that's the case, should we provide both the terminal-based option and the sandbox option?

Reply:
> Is it people looking at our repo on github.com? 

Yes. Even though I think we can assume more technical folks check out Fleet on GitHub, I propose removing the terminal-based option because Fleet sandbox is now the recommended way to try Fleet.

`fleetctl preview` will transition to a tool used by Fleet contributors to test changes to Fleet.

This means, that if any bugs are discovered that break `fleetctl preview` may take up to 3 weeks to fix (bugs will be prioritized in the current release instead of cutting a patch). This means, if there's a `fleetctl preview` bug, it's possible folks trying Fleet using the terminal-based option will see a broken Fleet.

Instead, patch releases will be cut for any bugs that break Fleet sandbox.